libqtile/layout/columns.py

```diff
@@ -118,7 +118,7 @@ class Columns(Layout):
         ("border_width", 2, "Border width."),
         ("border_on_single", False, "Draw a border when there is one only window."),
         ("margin", 0, "Margin of the layout (int or list of ints [N E S W])."),
-        ("margin_on_single", False, "Add margin to the layout for only one window."),
+        ("margin_on_single", -1, "Margin when only one window. `-1` means use `margin`."),
```

Comment:
Let's keep consistent for users that already use a margin:

```suggestion
        ("margin_on_single", 0, "Margin when only one window. `-1` means use `margin`."),
```

Reply:
Well the setting was only introduced to master 30 minutes before this PR was submitted, and that commit itself changed the default setting for everyone by making `margin_on_single` equal false, where before single windows used `margin` and therefore did have margins if there were any. So `-1` restores the default of using `margin`'s size for single windows.

Reply:
Wut? If I have margin=10 from an older version of Qtile and get this with -1 as default, I'll have a margin on single windows when I didn't have one before.

Reply:
My point being that no, Columns didn't use the margin for single windows before.

Reply:
My bad, it did. :)